### DISCUSSION TITLE
Everything Suddenly .Vid

### DISCUSSION THREAD

**Discussion by Mura2099** [aclap-dev/video-downloadhelper#118] [Help and Q&A]
### Your question

Suddenly everything is just a string of alphanumerics and in the .vid format (ie `65cf52acd6aac.vid`). I think the software updated in the background; I did nothing, though. This format is not even playable, so currently the entire plugin has effectively stopped working.

### Addon & CoApp version and details

8.2.0.23 (stable)
VdhCoApp 2.0.17
Windows 10
Firefox

**Comment by paulrouget** [comment]
From what website, and how do you download exactly?

**Reply by Mura2099** [reply]
From everywhere using any of the download methods. Reverted to 8.1.4.0 and the problem instantly went away without doing anything else. So the new version is just outright broken, or changes some default setting in odd ways that result in it being broken (I have never had to change any settings, so everything is default).

**Reply by paulrouget** [reply]
Sorry, I need more details.

Give at least one link that's failing for you.

Also, copy/paste these information, I need that to help you: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Reply by paulrouget** [reply]
@Mura2099 is it a HLS or Dash website?

**Reply by paulrouget** [reply]
I have the feeling that might be a issue with in browser download.

Can you try see if the beta version fixes your issue: https://www.downloadhelper.net/beta

**Reply by Mura2099** [reply]
Here is one site that has examples (https://sfdebris.com/videos/startrek/e103.php). Since I have reverted to the prior release I can't give you the info paste.

**Reply by paulrouget** [reply]
Yeah, that's what I expected. These are non-HLS videos downloaded via the browser.

We fixed that.

You can wait a couple of days until the new version is available, or you can install the beta here: https://www.downloadhelper.net/beta
